spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -133,82 +135,97 @@ public void setIntegrationEvaluationContext(EvaluationContext evaluationContext)
 
 	@Override
 	public final void afterPropertiesSet() throws Exception {
-		Assert.notNull(this.applicationContext, "The 'applicationContext' property must not be null");
+		Assert.notNull(this.applicationContext,
+				"The 'applicationContext' property must not be null");
 		if (this.evaluationContext == null) {
-			this.evaluationContext = ExpressionUtils.createStandardEvaluationContext(getBeanFactory());
+			this.evaluationContext = ExpressionUtils
```

Comment:
Yes, that's something we should enforce as well I think. Spring Framework rules are 120 max for lines and 80 max for Javadoc.